File: fuzzer/compiled_programs/gen_NHdEQkmRXYVl.py
from sklearn import *
import pandas as pd

YxHQ = pd.read_csv("../datasets/Iris__Species.csv")
YxHQ = YxHQ.dropna()

YxHQ_Y = YxHQ["Species"]
YxHQ = YxHQ.drop(columns=["PetalWidthCm", "Species"])

hvwvhVfU = neighbors.KNeighborsClassifier()

YxHQ_X_train, YxHQ_X_test, YxHQ_Y_train, YxHQ_Y_test = model_selection.train_test_split(YxHQ, YxHQ_Y, test_size = 0.3071165972922678)
hvwvhVfU.fit(YxHQ_X_train, YxHQ_Y_train)

print("Accuracy score : " + str(metrics.accuracy_score(YxHQ_Y_test, hvwvhVfU.predict(YxHQ_X_test))))

YxHQ_X_train, YxHQ_X_test, YxHQ_Y_train, YxHQ_Y_test = model_selection.train_test_split(YxHQ, YxHQ_Y, test_size = 0.9502723340332941)
hvwvhVfU.fit(YxHQ_X_train, YxHQ_Y_train)

print("Accuracy score : " + str(metrics.accuracy_score(YxHQ_Y_test, hvwvhVfU.predict(YxHQ_X_test))))

YxHQ_X_train, YxHQ_X_test, YxHQ_Y_train, YxHQ_Y_test = model_selection.train_test_split(YxHQ, YxHQ_Y, test_size = 0.897970409331531)
hvwvhVfU.fit(YxHQ_X_train, YxHQ_Y_train)

print("Accuracy score : " + str(metrics.accuracy_score(YxHQ_Y_test, hvwvhVfU.predict(YxHQ_X_test))))

File: fuzzer/compiled_programs/gen_NHdEQkmRXYVl.r
library(tidyr)
suppressMessages(library(dplyr))
library(e1071) # SVM implementation
library(class) # KNN implementation
library(rpart) # DecisionTree implementation
library(nnet) # MLP implementation
suppressMessages(library(caret))

YxHQ <- read.csv("../datasets/Iris__Species.csv")
YxHQ <- YxHQ %>% drop_na()

columns_to_drop <- c("PetalWidthCm")
YxHQ <- YxHQ %>% select(-any_of(columns_to_drop))

YxHQ_X <- subset(YxHQ, select = -Species)
YxHQ_Y <- as.factor(YxHQ$Species)

train_index <- sample(1:nrow(YxHQ), (1-0.3071165972922678) * nrow(YxHQ))
YxHQ_X_train <- YxHQ_X[train_index, ]
YxHQ_X_test <- YxHQ_X[-train_index, ]
YxHQ_Y_train <- YxHQ_Y[train_index]
YxHQ_Y_test <- YxHQ_Y[-train_index]

hvwvhVfU <- knn(train = YxHQ_X_train, test = YxHQ_X_test, cl = YxHQ_Y_train)

hvwvhVfU_accuracy <- confusionMatrix(hvwvhVfU, YxHQ_Y_test)$overall["Accuracy"]
print(paste("hvwvhVfU accuracy:", hvwvhVfU_accuracy))

train_index <- sample(1:nrow(YxHQ), (1-0.9502723340332941) * nrow(YxHQ))
YxHQ_X_train <- YxHQ_X[train_index, ]
YxHQ_X_test <- YxHQ_X[-train_index, ]
YxHQ_Y_train <- YxHQ_Y[train_index]
YxHQ_Y_test <- YxHQ_Y[-train_index]

hvwvhVfU <- knn(train = YxHQ_X_train, test = YxHQ_X_test, cl = YxHQ_Y_train)

hvwvhVfU_accuracy <- confusionMatrix(hvwvhVfU, YxHQ_Y_test)$overall["Accuracy"]
print(paste("hvwvhVfU accuracy:", hvwvhVfU_accuracy))

train_index <- sample(1:nrow(YxHQ), (1-0.897970409331531) * nrow(YxHQ))
YxHQ_X_train <- YxHQ_X[train_index, ]
YxHQ_X_test <- YxHQ_X[-train_index, ]
YxHQ_Y_train <- YxHQ_Y[train_index]
YxHQ_Y_test <- YxHQ_Y[-train_index]

hvwvhVfU <- knn(train = YxHQ_X_train, test = YxHQ_X_test, cl = YxHQ_Y_train)

hvwvhVfU_accuracy <- confusionMatrix(hvwvhVfU, YxHQ_Y_test)$overall["Accuracy"]
print(paste("hvwvhVfU accuracy:", hvwvhVfU_accuracy))

